tests/ui/pages/data_entity/data_entity.page.ts

```diff
@@ -0,0 +1,57 @@
+import BasePage from '../base-page';
+
+const SELECTORS = {
+  addOwner: `text=Add Owner`,
+  inputName: `[placeholder="Search name"]`,
+  inputRole: `[placeholder="Search title"]`,
+  autocomplete: text => `[role="presentation"]:has-text('${text}')`,
+  addOwnerButton: `[type="submit"]`,
+  addTag: `.MuiButton-root.MuiButton-text.MuiButton-textPrimary.MuiButton-sizeSmall:has-text('Add Tags')`,
+  inputTagName: `[role="combobox"]`,
+  saveButton: `[type="submit"]`,
+  ownerString: text => `.sc-dxtBLK.hWlgkE:has-text('${text}')`,
+  tagString: text => `.MuiTypography-root.MuiTypography-body1.sc-jdAMXn:has-text('${text}')`,
+    tag: text => `p:has-text('${text}')`,
+    addGroup: `.MuiButtonBase-root.MuiButton-root.MuiButton-text.MuiButton-textPrimary.MuiButton-sizeSmall:has-text('Add to group')`
+};
+
+export default class DataEntityPage extends BasePage {
+  async openAddOwnerModal() {
+    await this.page.locator(SELECTORS.addOwner).click();
+  }
+
+  async fillInputOwnerName(name: string, text: string) {
+    await this.page.locator(SELECTORS.inputName).fill(name);
+    await this.page.locator(SELECTORS.autocomplete(text)).click();
+  }
+
+  async fillInputOwnerRole(name: string, text: string) {
+    await this.page.locator(SELECTORS.inputRole).fill(name);
+    await this.page.locator(SELECTORS.autocomplete(text)).click();
+  }
+
+  async createOwner(name: string, textName: string, title: string, textTitle: string) {
```

Comment:
I opened the "Add owner" modal (by clicking the button) and I see only 2 input fields there.

Why do we need 4 arguments in this method?

Reply:
because when we fill in these inputs the first time, we have to create the owner and title, and we can make it by clicking on the text in select